moduleframework/mtf_scheduler.py

```diff
@@ -248,6 +250,35 @@ def avocado_general(self, avocado_default_args=[]):
             rc = cpe.returncode
         return rc
 
+    def _tcinfo(self, testcases, header, logs=True, description=True):
+        if testcases:
+            emptydelimiter = ""
+            harddelimiter = "------------------------"
+            common.print_info(emptydelimiter, "{0} {1} {0}".format(harddelimiter, header))
+            for testcase in testcases:
+                tcname = testcase.get('id').split("-", 1)[1]
```

Comment:
What about if 'id' is missing of field does not container two fields? Some except would be fine. If it make sense

Reply:
exactly. I had to do there exception handling in case something goes wrong